physics/continuous_trajectory.hpp

```diff
@@ -140,27 +142,26 @@ class ContinuousTrajectory : public Trajectory<Frame> {
   // stored in this vector sorted by their |t_max|, as it turns out that we
   // never need to extract their |t_min|.  Logically, the |t_min| for a
   // polynomial is the |t_max| of the previous one.  The first polynomial has a
-  // |t_min| which is |*first_time_|.
+  // |t_min| which is |*first_time_|.  The pointer is owned by |arena_|;
```

Comment:
`*polynomial` is owned by `arena_` (so is `polynomial` in that this struct is allocated there by the vector, but that's not the intent of that comment).